obspy/io/quakeml/tests/test_quakeml.py

```diff
@@ -881,7 +881,18 @@ def test_write_with_extra_tags_and_read(self):
              'tX': {'value': UTCDateTime('2013-01-02T13:12:14.600000Z'),
                     'namespace': r'http://test.org/xmlns/0.1'},
              'dataid': {'namespace': r'http://anss.org/xmlns/catalog/0.1',
-                        'type': 'attribute', 'value': '00999999'}})
+                        'type': 'attribute', 'value': '00999999'},
+             # some nested tags :
+             'quantity': {'namespace': r"http://some-page.de/xmlns/1.0",
```

Comment:
Using `r"..."` is a bit strange here. Why is that done? But I see that it is also used in the other test cases so am I missing something?